grammar.rst

```diff
@@ -49,6 +49,9 @@ Note: sometimes things mysteriously don't work.  Before giving up, try ``make cl
 * ``_Unparser`` in the :file:`Lib/ast.py` file may need changes to accommodate
   any modifications in the AST nodes.
 
+* If any AST node introduced or updated, changes needs to be reflacted to
+  :file:`Doc/library/ast.rst`. 
```

Comment:
Most of the checkpoints start with the name of the file, which makes them easier to scan. May I suggest:

```suggestion
* :file:`Doc/library/ast.rst` may need to be updated to reflect changes to AST nodes.
```

Reply:
Makes sense, thanks!